### PR TITLE
Burger's Blueshield Change (BBC)

### PR DESCRIPTION
**Pull request by BurgerLUA**
image

## About The Pull Request

Overhauls blueshield equipment significantly. They now get:
- A very formal and armored suit. Also comes with steel-toed dress shoes (basically reskinned jackboots). Has the same stats as current blueshield armor.
- A necktie that gives them Wrestling CQC when worn.
- Rocket Gloves that won't break your spine (unless you have a ton of bad quirks or something).
- Gun Holster with a slightly nerfed M1911 (25% damage reduction compared to the normal variant).
- Telebatong, pepper spray, flash.
- Box of flashbangs in the locker.
- Box of zipties in the locker.
- Pinpointer Pair kit in the locker. Basically two pinpointers that point at one another.
- Basic first aid kit.
- A funny lockbox briefcase that deals slightly more damage than a toolbox.

What was removed:
- Command Tier Paycheck. They now have the same paycheck as regular crew.
- The modsuit. Blueshields shouldn't get modsuits like this. As it stands, a lot of blueshields just wear the thing 24/7 and not during emergencies.
- The SMG. Not even the HoS has something as powerful as this roundstart.
- Blueshield Combat Medical Kit + Sensor Device. No more constantly monitoring the health of heads. This is a paramedic's job. Just listen to comms if they get hurt.
- Security belt + Stunbatong. You are not security.
- ~~Bowman headsets for blueshield. You are not security.~~ Ok fine you can have a little bit of security. Have to go to your locker, though :^)
- Security PDA + Security Emergency box. YOU ARE NOT SECURITY.

## Why It's Good For The Game

Blueshield are not security. They should never be security. Their entire job should be to take bullets meant for heads, while dishing out some of their own. The current aesthetic of blueshield is very questionable and is also very military. It legitimately needs to change into something more corporate and more "concealable" so blueshield isn't a job that military larpers take.

I am extremely tempted to remove the non-lethal equipment as well as existing clothing so I can drive home the point of blueshield being not security, but I have a feeling people won't like that.

## Proof Of Testing

See screenshot.

## Current Issues

There are no wrestling CQC icons for the moves. I don't know why. /tg/ issue, basically.

## Changelog

:cl: BurgerBB
balance: Overhauls blueshield equipment significantly.
/:cl:

### CODE REVIEW

**Comment by Majkl-J** [comment]
> Bowman headsets for blueshield. You are not security.

But.. you gave them flashbangs

**Comment by Alecksohs** [comment]
Let them keep their color scheme and clothing and we're good

**Comment by projectkepler-ru** [comment]
This is fucking terrible


I love it

**Comment by nevimer** [comment]
very good PR

We will test this soon, probably today

**Comment by Superhats** [comment]
> > Bowman headsets for blueshield. You are not security.
> 
> But.. you gave them flashbangs

Flashbangs without bowman seems counter-intuitive yeah.

**Comment by StrangeWeirdKitten** [comment]
Based PR.

My own personal take,
* Crew monitors tracking heads is fine. A lot of blueshield gameplay comes from the bluey bugging all the heads of staff to turn their sensors on so they can use their limited pocket monitor. I say let them keep it.
* Bowmans obviously.
* I see CQC. If it can stunlock and sleep you, I'd be very careful of having an item like this.

**Comment by StrangeWeirdKitten** [comment]
We're TMing this still.

**Comment by Cyprex** [comment]
> A necktie that gives them Wrestling CQC when worn.

I'm guessing you meant Wrestling Martial Art? CQC is a specific Martial Art available to Syndies.

**Comment by IgiariValkyr** [comment]
Yeah. Keep their medkit in the locker. It's not doing anything wrong and usually never gets used anyways other than the pinpointer.

**Comment by BurgerLUA** [comment]
> > A necktie that gives them Wrestling CQC when worn.
> 
> I'm guessing you meant Wrestling Martial Art? CQC is a specific Martial Art available to Syndies.

Wrestling martial art yes

**Comment by StrangeWeirdKitten** [comment]
Obviously will need sprites for the specific abilities.

**Comment by nevimer** [comment]
https://github.com/Bubberstation/Bubberstation/actions/runs/8545633567/job/23415077891?pr=1345#step:10:1765

**Comment by Arturlang** [comment]
Martial arts is cooler than having a busted ass SMG, but why wrestling? Doesn't make sense thematically

**Comment by BurgerLUA** [comment]
There is a funny bug that I think is a /tg/ issue where martial arts isn't getting deleted when the object is deleted, which is occurring on the loadout preview character. This is probably because never in the history of a /tg/ server has anyone added martial arts as a loadout item.

**Comment by Spartan2548** [comment]
Alright, some feedback after playing it.
-The lack of the portable blueshield monitor encourages BS players to stay on the bridge, glued to the sensors monitor, or buy one from medical to see who is alive and alright
-The pinpointer pair is kinda useless, since the ONLY situation you would use it, is if you knew a head of staff was being targeted, and then you would stay with them
-The tie is a good idea, however, Wrestling is too unbalanced, and secondly, it's not fireproof to any degree, meaning ANYTIME you get hit by fire, if you don't extinguish it in a few seconds, you lose the tie forever, I recommend swapping them with a pair of advanced combat gloves or a straight up krav maga skillchip
-With the PR adding a fully powered M1911, giving the BS a gimped version is a bit redundant since they will either buy one within 15 seconds of spawning in via the NT-IRN or will ask the Captain/AC for one
-I don't really understand the whole 'toolbox secure case' item, it's a pretty funny gimmick idea, but so far, has seen zero use by Blueshields
-Lastly, the assistant-ifying of the role. I understand that 'defunded crackhead BlueShield' was funny, but gotta remember, lore-wise they are, albeit low-ranking, asset protection agents, not some random maint assistant who was given a Mosin and told to protect the captain from nukes

In conclusion. It's going in a good direction, turning the Blueshield from 'spec ops military operative' aesthetic to one that anyone would think of whenever someone says bodyguard, but it's going a bit too far into the Assistant territory.

**Comment by Angoryskaven** [comment]
Having played with it, I can definitely say that this PR is not that good, for these specific reasons:

-It removes the ability to check in on your heads of staff, as comms is an extremely unreliable way of seeing if they got hurt; 

-It removes the MODsuit, which is useful and, imo needed. If a head of staff is being attacked in a spaced room, what are you supposed to do? Just stare as you cannot enter the room? ;

-Adds "Martial Arts" that make absolutely no sense. You can delimb people with a simple ability that requires you to grab them (something that the tie allows you to do fast and aggressively). You can kick someone into the stratosphere, dive bomb them or launch them with another grab ability;

-It removes the SMG for dubious reasons. It is cited as being ""strong"", yet for some reason, most, if not all Blueshields I saw exchanged it for a MCR.

**Comment by projectkepler-ru** [comment]
this is fucking terrible

I don't like it anymore

**Comment by AllCorruptingWyrm** [comment]
Hey did you know all you need is a passive grab and prepared clinch move to throw someone across the map/room and hard stun everyone around you?  That stun is like four seconds by the way and you could theoretically chain it. 

I have no idea why we insist on changing the job, this actually makes the balance worse and if I was in bad faith i'd make a video and clip it and show why its busted.

**Comment by StrangeWeirdKitten** [comment]
Admittidely while I love the moves
- Requires sprites
- A lot of them are really powerful, admittidely. I'm going to un-tm this and my other PR for the time being.

**Comment by AllCorruptingWyrm** [comment]
I was originally going to write out a long post about why this was a bad idea, but I got forcefully converted as Wireweed and managed to solo convert nearly the entire crew with wrestling. No, I'm not making an epic montage (because I'm not bad faith) but I think this just shows to show that all of these changes really aren't doing anything but making people upset, with that said, **please stop touching Blueshield.**

**Comment by coffeecoffeecoffee** [comment]
Hold on, now, though.

There's no need to throw the suit out with the bathwater.

**Comment by AllCorruptingWyrm** [comment]
okay, the suit is cool.

Please let that just be a loadout option.

**Comment by Redrover1760** [comment]
More choices towards blueshield aesthetics, as well as blueshield loadouts are great ideas. Maybe a more medical looking uniform would be pretty awesome.

**Comment by Arturlang** [comment]
This PR could also just be a round trait

**Comment by KazooBard** [comment]
My five cents is while wrestling and CQC is strong Krav Maga is just right. Yes, it makes warden's gloves less unique, but they never leave brig anyways.

The suit itself and the inspiration with tackler gloves is what I really, really would love to see ingame. Practical drip that's role accurate is something I really like to see more of.

**Comment by BurgerLUA** [comment]
image

*scream is this PR open or closed
